cli/src/complete.rs

```diff
@@ -110,6 +111,11 @@ fn get_jj_command() -> Result<std::process::Command, CommandError> {
     let args: GlobalArgs = GlobalArgs::from_arg_matches(&args)?;
 
     if let Some(repository) = args.repository {
+        // Try to update repo-specific config on a best-effort basis.
+        if let Ok(loader) = DefaultWorkspaceLoaderFactory.create(&cwd.join(&repository)) {
+            let _ = layered_configs.read_repo_config(loader.repo_path());
```

Comment:
nit: propagate config error consistently?

I think it's also good to suppress all `user`/`repo_config` errors.

Reply:
I suppressed these additional errors:
```rust
let maybe_cwd_workspace_loader = DefaultWorkspaceLoaderFactory.create(find_workspace_dir(&cwd));
let _ = layered_configs.read_user_config();
if let Ok(loader) = &maybe_cwd_workspace_loader {
    let _ = layered_configs.read_repo_config(loader.repo_path());
}
```
Let me know if there are more you think should be suppressed.